tests/init/neg/leak-warm.scala

```diff
@@ -0,0 +1,30 @@
+object leakWarm {
+    abstract class A(tag: Int) {
+        class B(x: Int) {
+            val y = x
+        }
+        def m(): B
+    }
+
+    class C(tag1: Int, tag2: Int) extends A(tag1) {
+        def m() = new B(5)
+    }
+
+    class D(tag1: Int, tag2: Int) extends A(tag1 + tag2) {
+        def m() = new B(tag1)
+    }
+    val c = new C(1, 2)
+    val d = new D(3, 4)
+<<<<<<< HEAD
+    val l: List[A] = List(c, d) // error
+=======
+    val l: List[A] = List(c, d) // error // error
+<<<<<<< HEAD
+=======
+    val l: List[A] = List(c, d) // error
+>>>>>>> 93ec7cb48e... Update leak-warm.scala
+=======
+>>>>>>> 5e4f243160... re-organize tests
+>>>>>>> Add tests
+    val l2 = l.map(_.m())
```

Comment:
This file needs to be updated because of merge conflicts.

Reply:
I've fixed that